src/init.c

```diff
@@ -680,7 +680,7 @@ static void jl_resolve_sysimg_location(JL_IMAGE_SEARCH rel)
     if (jl_options.output_code_coverage)
         jl_options.output_code_coverage = absformat(jl_options.output_code_coverage);
     if (jl_options.tracked_path)
-        jl_options.tracked_path = absformat(jl_options.tracked_path);
+        jl_options.tracked_path = abspath(jl_options.tracked_path, 0);
```

Comment:
I think this was a mistake in https://github.com/JuliaLang/julia/pull/44359
`absformat` doesn't seem to absolutify a relative path.

Reply:
Maybe split those changes out? We should probably backport them.